locale/en-us.yaml

```diff
@@ -1729,9 +1729,11 @@ lock_version_mismatch:
   other: This project is locked at State Tool version [NOTICE]{{.V0}}[/RESET]. Your current State Tool version is [NOTICE]{{.V1}}@{{.V2}}[/RESET].
 lock_update_lock:
   other: You can lock the project to the running State Tool version with `[ACTIONABLE]state update lock[/RESET]`.
+err_learn_open:
+  other: "Could not open cheat sheet in browser, visit: [ACTIONABLE]{{.V0}}[/RESET] to learn about common commands"
```

Comment:
Since we'll already print the "Opening .." message we can probably drop the URL here so we don't stutter.